awslabs/solution-spark-on-aws
# fix: add correct HTTP status codes for start/stop request

Issue #, if available:

Description of changes:
* Cannot start an environment while it's being stopped
* Cannot stop an environment while it's being started
* If starting an environment while it's being started return a 204
* If stopping an environment while it's being stopped return a 204
* Add integration for start/stop when environment does not exist

**Testing**
* Manual tests passed of start/stop API passed
* Integration tests passed


Checklist:




By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
